tools/accuracy_checker/openvino/tools/accuracy_checker/launcher/dlsdk_launcher.py

```diff
@@ -931,7 +931,10 @@ def _set_preprocess(self, preprocess):
         self.disable_resize_to_input = preprocess.ie_processor.has_resize()
 
     def get_model_file_type(self):
-        return self._model.suffix
+        if self._delayed_model_loading:
```

Comment:
what if initially model loading is postponed, but italready loaded at the moment when we call interface, probably we should check self._model itself for this case instead

Reply:
something like:
```suggestion
        if hasattr(self, '_model'):
```

Reply:
Done